tools/pytorch2onnx.py

```diff
@@ -96,13 +96,16 @@ def _prepare_input_img(img_path,
     return mm_inputs
 
 
-def _update_input_img(img_list, img_meta_list):
+def _update_input_img(img_list, img_meta_list, update_origin=False):
```

Comment:
What does `update_origin` means?

Reply:
`ori_shape` in img_meta is used to rescale output to origin shape.
`_update_input_img` will update `img_meta_list` according to `img_list`. And this flag is used to control if the segmentator needs to rescale according to the input tensor shape.